File: Real estate recommendation Platform/Front-end/User Front-end/FindMyHome/src/app/model/agency.ts
export class Agency
{
    id:number|undefined;
    agencyId:string|undefined;
    agencyName:string|undefined;
    agencyAddress:string|undefined;
    agencyPhoneNumber:string|undefined;

    constructor(agencyId:string,agencyName:string,agencyAddress:string,agencyPhoneNumber:string,id:number){

        this.id = id;
        this.agencyId = agencyId;
        this.agencyName = agencyName;
        this.agencyAddress = agencyAddress;
        this.agencyPhoneNumber = agencyPhoneNumber;

    }

}

File: Real estate recommendation Platform/Back-end/Property.Admin/Property.Admin/Agency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Properties.Admin
{
    public class Agency : BaseObject
    {
        public string AgencyId { get; set; }
        public string AgencyName { get; set; }
        public string AgencyPhoneNumber { get; set; }
        public string AgencyAddress { get; set; }
    }
}
